app/src/main/kotlin/com/antosik/benchproject/app/common/binding/imageView/ImageViewBindingAdapters.kt

```diff
@@ -10,3 +11,10 @@ fun ImageView.loadMovieImage(imageUrl: String?) {
         .load(imageUrl)
         .into(this)
 }
+
+@BindingAdapter("isFavoriteChangeTint")
+fun ImageView.changeColor(isFavorite: Boolean) {
```

Comment:
Instead of binding adapter you could hold color id in your UI model `Movie`

Reply:
Done 😄